ext-apiserver/pkg/controller/config/configuration.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"github.com/spf13/pflag"
+)
+
+type Configuration struct {
+	Kubeconfig   string
+	Cloud        string
+	KubeadmToken string
+}
+
+var Test1 string = "hello"
+
+func GetTestString() string {
```

Comment:
What is the purpose of this logic on getting some test string?

Reply:
Oops. Fixed